api/v1alpha1/dspipeline_types.go

```diff
@@ -107,7 +107,9 @@ type APIServer struct {
 	// Default: true
 	// +kubebuilder:default:=true
 	// +kubebuilder:validation:Optional
-	AutoUpdatePipelineDefaultVersion bool `json:"autoUpdatePipelineDefaultVersion"`
+	AutoUpdatePipelineDefaultVersion bool   `json:"autoUpdatePipelineDefaultVersion"`
+	KFPLauncherImage                 string `json:"kfpLauncherImage,omitempty"`
+	DriverImage                      string `json:"driverImage,omitempty"`
```

Comment:
these are argo specific, I'm wondering if we should put this in a sub argo field like: 

```yaml
spec: 
  apiServer:
    ...
    V2Argo:
      KFPLauncherImage: ...
      DriverImage: ...

```

@gregsheremeta  / @gmfrasca  thoughts?

Reply:
I'm not sure if nesting this would add some unnessesary complexity but I do like the idea of denoting they're Argo related.  Perhaps naming them `ArgoLauncherImage` and `ArgoDriverImage` would be a good compromise?

Reply:
another consideration for @HumairAK 's suggestion would be that we would need to/should move the Tekton-specific images ie (MoveResultsImage) to their own sub-item

Reply:
I like both of Giulio's suggestions. Nothing else to add.

Reply:
let's just go with Argo* naming and not do the nesting